app/components/sign-up/get-started.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="landing">
+  <h2 class="landing__mainHeading">
+    {{this.mainHeading}}
+  </h2>
+  <h3 class="landing__subHeading">
+    {{this.subHeading}}
+  </h3>
+  {{#if (eq @state "get-started")}}
+    <ul class="landing__features-lists">
+      <li class="landing__features-list">Use Features</li>
+      <li class="landing__features-list">Display yourself on the (members) page</li>
+    </ul>
+  {{/if}}
+
+  <div class="get-started__btn">
+    <SignUp::Button
+      @clickHandler={{@changeRouteParams}}
+      @state={{@state}}
+      @disabled={{false}}
```

Comment:
NIT: I think we can pass this as a direct boolean (without the `{{ }}`)

Reply:
It wont work if we pass without `{{ }}`